domain/src/main/java/org/oppia/android/domain/topic/PrimeTopicAssetsController.kt

```diff
@@ -1,18 +0,0 @@
-package org.oppia.android.domain.topic
-
-// TODO(#169): Remove this controller & download priming once downloads are properly supported.
```

Comment:
Update docs.

Also, explain removal of this controller & its impl, plus the annotations.

Reply:
Done.